src/components/tou-modal/variations/tou-last-reminder-under-16.jsx

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable react/jsx-no-bind */
+import React, {useState} from 'react';
+import Modal from '../../modal/base/modal.jsx';
+import {FormattedMessage, injectIntl} from 'react-intl';
+import {TermsOfUseLink} from '../tou-modal.jsx';
+import PropTypes from 'prop-types';
+import {TosEmailSentStep} from './tou-under-16.jsx';
+
+// eslint-disable-next-line arrow-body-style
+const TosModalLastReminderUnder16 = () => {
+    const [currentStep, setCurrentStep] = useState(1);
+    const handleNextStep = () => {
+        setCurrentStep(prevStep => prevStep + 1);
+    };
+
+    return (
+        <Modal
+            overlayClassName="tou-modal-overlay"
+            className="tou-modal"
+            showCloseButton={currentStep === 2}
+            isOpen
+        >
+            <div className="tou-modal-top" />
+            <div className="tou-modal-content">
+                {currentStep === 1 ? (
+                    <Step1 onNextStep={handleNextStep} />
+                ) : (
+                    <TosEmailSentStep />
+                )}
+            </div>
+        </Modal>
+    );
+};
+
+// eslint-disable-next-line arrow-body-style
+const Step1 = ({onNextStep}) => {
```

Comment:
Can we change the name of the component to something more explanatory? Same for `src/components/tou-modal/variations/tou-reminder-under-16.jsx` and `src/components/tou-modal/variations/tou-under-16.jsx` etc